include/gridtools/storage/data_store.hpp

```diff
@@ -10,6 +10,7 @@
 #pragma once
 
 #include <memory>
+#include <numeric>
```

Comment:
is it used here?

Reply:
I checked the whole file -- `<numeric>` is not used, but `<utility>` is used (`std::move`).

Reply:
ups, I fixed it but didn't save the editor